### ISSUE TITLE
✅ 🐞 Bug Report: Menu widget does not include all breakpoint options in Responsive Settings - [ED-12111]

### ISSUE THREAD

**Issue by simplicity17** [elementor/elementor#22488]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I'm building out a site and setup four breakpoints in Elementor, the three default ones (Desktop, Tablet, Mobile) and one additional, Tablet Landscape, for smaller computer screens. The new menu widget works great for creating mega menus, however in the Responsive setting tab only the default breakpoint options, Tablet Portrait and Mobile Portrait, appear! There is no way to set the breakpoint to Tablet Landscape, Mobile Landscape, etc. which makes it impossible to achieve the responsiveness necessary on the site. Since this should be a feature by default given that breakpoints are not a beta feature, I'm reporting this as a bug instead of a feature request.

### Steps to reproduce

1) Create additional breakpoints in Elementor Site Settings
2) Insert the new Menu widget into the header template of the website and build it out
3) Click on the Responsive Settings tab in Elementor editor for the Menu widget and note that there are only 2 breakpoint options instead of 4 (the 4th should be "none" like in the original Wordpress Menu widget).

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 84.1, Revision b308619 v5.6.41-84.1
	PHP Version: 7.4.33
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 516M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.1
	Site URL: https://dev.canfp.org
	Home URL: https://dev.canfp.org
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%year%/%postname%/
	Language: en-US
	Timezone: America/Los_Angeles
	Admin Email: emily@simplicitydesign.business
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Editor Tools
		Version: 5.9.0
		Author: Automattic

	Basic User Avatars
		Version: 1.0.7
		Author: Stranger Studios

	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Code Snippets
		Version: 3.3.0
		Author: Code Snippets Pro

	Duplicate Page
		Version: 4.5.2
		Author: mndpsingh287

	Elementor
		Version: 3.13.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.1
		Author: Elementor.com

	JetEngine
		Version: 3.1.5
		Author: Crocoblock

	JetPopup
		Version: 2.0.0
		Author: Crocoblock

	JetSmartFilters
		Version: 3.1.1
		Author: Crocoblock

	Paid Memberships Pro
		Version: 2.10.5
		Author: Paid Memberships Pro

	Paid Memberships Pro - Approvals Add On
		Version: 1.5
		Author: Stranger Studios

	Paid Memberships Pro - Auto-Renewal Checkbox
		Version: 0.3.0
		Author: Paid Memberships Pro

	Paid Memberships Pro - AWeber Add On
		Version: 1.3.2
		Author: Paid Memberships Pro

	Paid Memberships Pro - Cancel on Next Payment Date
		Version: 0.5.1
		Author: Paid Memberships Pro

	Paid Memberships Pro - Donations
		Version: 1.1.1
		Author: Paid Memberships Pro

	Paid Memberships Pro - Invite Only Add On
		Version: .3.4
		Author: Stranger Studios

	Paid Memberships Pro - Member Badges Add On
		Version: 1.0
		Author: Paid Memberships Pro

	Paid Memberships Pro - Membership Maps Add On
		Version: 0.5
		Author: Paid Memberships Pro

	Paid Memberships Pro - Reason For Cancelling Add On
		Version: 0.2.1
		Author: Paid Memberships Pro

	Paid Memberships Pro - Recurring Emails Add On
		Version: 0.5.5
		Author: Stranger Studios, Thomas Sjolshagen <thomas@eighty20results.com>

	Paid Memberships Pro - Roles Add On
		Version: 1.4.2
		Author: Paid Memberships Pro

	Paid Memberships Pro - Sponsored Members Add On
		Version: 0.10.1
		Author: Paid Memberships Pro

	Paid Memberships Pro - Variable Pricing Add On
		Version: 0.4.5
		Author: Paid Memberships Pro

	Redirection
		Version: 5.3.10
		Author: John Godley

	Site Kit by Google
		Version: 1.99.0
		Author: Google

	Temporary Login Without Password
		Version: 1.8.1.1
		Author: StoreApps

	The HostGator Plugin
		Version: 1.2.8
		Author: HostGator

	UpdraftPlus - Backup/Restore
		Version: 2.23.4.1
		Author: UpdraftPlus.Com, DavidAnderson

	WooCommerce
		Version: 7.7.0
		Author: Automattic

	WP Bouncer - Limit Simultaneous Logins
		Version: 1.5.1
		Author: Stranger Studios

	WP Ultimate CSV Importer
		Version: 7.9
		Author: Smackcoders

	Yoast SEO
		Version: 20.7
		Author: Team Yoast


== Must-Use Plugins ==
	Endurance Page Cache
		Version: 2.2
		Author: Mike Hansen


== Features ==
	Custom Fonts: 0
	Custom Icons: 1

== Integrations ==
	
	woocommerce: Active


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Active
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Active


== Log ==
	
Log: showing 10 of 102023-05-13 13:24:50 [info] elementor::elementor_updater Started 
2023-05-13 13:24:50 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-13 13:24:50 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-13 13:24:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.1',
  'to' => '3.13.2',
)]
2023-05-15 15:35:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-15 15:35:41 [info] elementor-pro::elementor_pro_updater Started 
2023-05-15 15:35:41 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-15 15:35:41 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-15 15:35:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-15 15:35:43 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]

PHP: showing 6 of 6PHP: 2023-05-15 15:34:45 [notice X 963][/home1/canfp/dev.canfp.org/wp-content/plugins/elementor-pro/modules/woocommerce/module.php::381] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-15 17:37:10 [notice X 20][/home1/canfp/dev.canfp.org/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-15 17:37:21 [notice X 21][/home1/canfp/dev.canfp.org/wp-content/plugins/elementor/core/dynamic-tags/manager.php::73] Array to string conversion [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-16 07:39:24 [error X 12][/home1/canfp/dev.canfp.org/wp-content/plugins/elementor/includes/managers/controls.php::615] Maximum execution time of 60 seconds exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-16 19:20:07 [error X 1][/home1/canfp/dev.canfp.org/wp-content/plugins/elementor-pro/modules/payments/module.php::142] Uncaught Error: Call to undefined method ElementorPro\Core\Utils::_unstable_check_document_permissions() in /home1/canfp/dev.canfp.org/wp-content/plugins/elementor-pro/modules/payments/module.php:142
Stack trace:
#0 [internal function]: ElementorPro\Modules\Payments\Module->get_stripe_tax_rates(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /home1/canfp/dev.canfp.org/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home1/canfp/dev.canfp.org/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /home1/canfp/dev.canfp.org/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#4 /home1/canfp/dev.canfp.org/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /home1/canfp/dev.canfp.org/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-16 19:26:31 [notice X 1][/home1/canfp/dev.canfp.org/wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 20 of 26JS: 2023-05-16 16:48:53 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view37483") has already been destroyed and cannot be used. 
JS: 2023-05-16 16:58:23 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view12455") has already been destroyed and cannot be used. 
JS: 2023-05-16 16:59:53 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view14610") has already been destroyed and cannot be used. 
JS: 2023-05-16 17:00:07 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view19060") has already been destroyed and cannot be used. 
JS: 2023-05-16 17:00:35 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view22560") has already been destroyed and cannot be used. 
JS: 2023-05-16 17:15:45 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view81732") has already been destroyed and cannot be used. 
JS: 2023-05-16 17:42:57 [error X 3][https://dev.canfp.org/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.13.1:2:63893] Cannot convert undefined or null to object 
JS: 2023-05-16 18:22:15 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view15565") has already been destroyed and cannot be used. 
JS: 2023-05-16 18:59:38 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view15700") has already been destroyed and cannot be used. 
JS: 2023-05-16 19:14:10 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view13748") has already been destroyed and cannot be used. 
JS: 2023-05-16 19:14:17 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view17426") has already been destroyed and cannot be used. 
JS: 2023-05-16 19:29:11 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view64021") has already been destroyed and cannot be used. 
JS: 2023-05-16 19:36:43 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view2969") has already been destroyed and cannot be used. 
JS: 2023-05-17 00:31:20 [error X 3][https://dev.canfp.org/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.2:3:794943] Cannot read properties of undefined (reading 'toLowerCase') 
JS: 2023-05-17 00:53:35 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view9280") has already been destroyed and cannot be used. 
JS: 2023-05-17 00:53:35 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view10850") has already been destroyed and cannot be used. 
JS: 2023-05-17 01:40:59 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view5927") has already been destroyed and cannot be used. 
JS: 2023-05-17 01:57:41 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view18119") has already been destroyed and cannot be used. 
JS: 2023-05-17 02:01:33 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view11003") has already been destroyed and cannot be used. 
JS: 2023-05-18 16:37:38 [error X 1][https://dev.canfp.org/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1709") has already been destroyed and cannot be used. 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @simplicity17! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by lensbreak** [comment]
+1

**Comment by 319heads** [comment]
+1 

Can we please get an update on this bug? The new menu widget is currently useless without the ability to make it properly responsive.

**Comment by nicholaszein** [comment]
Hey there everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.18.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
